lib/store.js

```diff
@@ -60,7 +60,7 @@ class Store {
       // TODO : Await for callback
       setTimeout(() => {
         this.ws.send(JSON.stringify(cpuChoiceObj));
-      }, 1000);
+      }, 500);
```

Comment:
Hrm. I just noticed that this causes the ws.send() to happen in the background, half a second after this async function returns. I don't expect that this will break anything, but it is super-surprising (for example, what happens if someone deletes this.ws in that half-second window? Does the error get propagated up, or does the main play loop try to carry on regardless?).

I would suggest following this example: https://stackoverflow.com/a/39914235 : 

Make a global helper:
```
function sleep(ms) {
  return new Promise(resolve => setTimeout(resolve, ms));
}
```

and then call:
```
await sleep(500);
this.ws.send(JSON.stringify(cpuChoiceObj));
```

The practice of avoiding forking things off into the background, and always awaiting things is called "structured concurrency".